<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sensor_act">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_FrontDoor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Front Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Door1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Door2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit0">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="addAuthName">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>16</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addAuthFalse">
    <property name="geometry">
     <rect>
      <x>782</x>
      <y>40</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="addAuthTrue">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="textBrowserRefresh">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="idCardBrowser">
    <property name="geometry">
     <rect>
      <x>842</x>
      <y>286</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="idCardBrowserRefresh">
    <property name="geometry">
     <rect>
      <x>1082</x>
      <y>286</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idCardId">
    <property name="geometry">
     <rect>
      <x>858</x>
      <y>256</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="idCardFalse">
    <property name="geometry">
     <rect>
      <x>1049</x>
      <y>255</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="idCardTrue">
    <property name="geometry">
     <rect>
      <x>987</x>
      <y>255</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Font Awesome 6 Free</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idCardAddress">
    <property name="geometry">
     <rect>
      <x>987</x>
      <y>214</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idCardName">
    <property name="geometry">
     <rect>
      <x>857</x>
      <y>214</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>198</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>989</x>
      <y>198</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>240</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
